wscript

```diff
@@ -127,10 +137,13 @@ def configure(cfg):
 
     cfg.env.BOARD = cfg.options.board
     cfg.env.DEBUG = cfg.options.debug
+    cfg.env.AUTOCONFIG = cfg.options.autoconfig
```

Comment:
They are different environments. I thought about creating a function for the common variables, but in the end decided to leave as is..